addons/mail/static/src/models/rtc_session.js

```diff
@@ -408,9 +421,14 @@ registerModel({
          * sessions from other channels with the same partner (sessions opened from different
          * tabs or devices).
          */
-        rtc: one('Rtc', {
```

Comment:
this one is still the old name

https://github.com/odoo/odoo/blob/master/addons/mail/static/src/components/rtc_call_participant_card/rtc_call_participant_card.xml#L67